frontend/components/form/FormCheckbox.vue

```diff
@@ -1,11 +1,11 @@
 <template>
   <div class="relative flex flex-row items-center justify-start mb-3">
     <input
+      class="cursor-pointer mb-0 peer appearance-none w-[1.375rem] h-[1.375rem] border border-black rounded-sm bg-light-button dark:bg-dark-button dark:border-dark-text"
       type="checkbox"
       v-bind="{ ...$attrs, onChange: updateValue }"
```

Comment:
Should `v-bind` also be above. I don't think we'll have many instances of multiple `v-` attributes.